src/coreclr/jit/importercalls.cpp

```diff
@@ -1485,6 +1542,35 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         impPushOnStack(call, tiRetVal);
     }
 
+#ifdef SWIFT_SUPPORT
+    if (swiftErrorIndex >= 0)
+    {
+        // This Swift call uses the error register
+        assert(call->IsCall());
+        assert(call->AsCall()->unmgdCallConv == CorInfoCallConvExtension::Swift);
+        assert(swiftErrorIndex < (int)sig->numArgs);
+        CallArg* errorArg = call->AsCall()->gtArgs.GetArgByIndex(swiftErrorIndex);
+
+        assert(errorArg != nullptr);
+        GenTree* argNode = errorArg->GetNode();
+        assert(argNode != nullptr);
+
+        // SwiftError* arg should have been spilled to a local temp variable
+        assert(argNode->OperIs(GT_LCL_VAR));
+
+        // Store the error register value to where the SwiftError* points to
+        GenTree* errorRegNode = new (this, GT_SWIFT_ERROR) GenTree(GT_SWIFT_ERROR, TYP_REF);
+        errorRegNode->SetHasOrderingSideEffect();
+
+        GenTree*         argNodeCopy     = gtNewLclvNode(argNode->AsLclVar()->GetLclNum(), argNode->TypeGet());
+        GenTreeStoreInd* swiftErrorStore = gtNewStoreIndNode(argNodeCopy->TypeGet(), argNodeCopy, errorRegNode);
+        impAppendTree(swiftErrorStore, CHECK_SPILL_ALL, impCurStmtDI, false);
+
+        // Indicate the error register will be checked after this call returns
+        call->AsCall()->gtCallMoreFlags |= GTF_CALL_M_SWIFT_ERROR_HANDLING;
+    }
+#endif // SWIFT_SUPPORT
```

Comment:
Perhaps introduce a method for this just to avoid growing `impImportCall` even more